test/docs/sotd/rec-publish-corrections-fail2.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/W3C-REC">
+</head>
+
+<body>
+    <div class="head">
+        <a class="logo" href="https://www.w3.org/"><img alt="W3C" src="https://www.w3.org/StyleSheets/TR/2016/logos/W3C" width="72" height="48"></a>
+        <h1 id="title" class="title p-name">High Resolution Time</h1>
+
+        <h2>
+            W3C Recommendation
+            <time class="dt-published" datetime="2020-09-15">15 September 2020</time>
+        </h2>
+    </div>
+    <section id="sotd" data-link-for="Performance" class="introductory">
+        <h2>Status of This Document</h2>
+        <!-- fail check sotd.publish with error "url-text-not-found" for having wrong text for <a>. The text should include words "proposed corrections" not "includes proposed" -->
+        <p>
+            This document was published by the <a href="https://www.w3.org/webperf/">Web Performance Working Group</a> as a Recommendation.
+            <!-- substantiveChanges: true, newFeatures: false -->
+            It <a href="https://www.w3.org/2020/Process-20200915/#proposed-corrections">includes proposed</a> corrections.
```

Comment:
It would probably be better to test the case @plehegar  is trying to fix

```suggestion
            It includes <a href="https://www.w3.org/2020/Process-20200915/#proposed-corrections">proposed correction</a>.
```

Reply:
plural will actually passed